bootstrap/kubeadm/internal/cloudinit/cloudinit_test.go

```diff
@@ -72,13 +75,13 @@ func TestNewInitControlPlaneAdditionalFileEncodings(t *testing.T) {
       hi`,
 	}
 	for _, f := range expectedFiles {
-		if !bytes.Contains(out, []byte(f)) {
-			t.Errorf("%s\ndid not contain\n%s", out, f)
-		}
+		g.Expect(bytes.Contains(out, []byte(f))).To(BeTrue())
```

Comment:
```suggestion
		g.Expect(bytes).To(ConsistOf([]byte(f)))
```
Although, I do wonder if it might be better to use `BeEquivalentTo` here instead, because I suspect we care about order here.

Reply:
per slack discussion: using `g.Expect(out).To(ContainElements([]byte(f)))`

Reply:
updated: using `g.Expect(string(out)).To(ContainSubstring(f))` now :)